server/container/guice/blob/blob-api-guice/pom.xml

```diff
@@ -25,7 +25,7 @@
         <groupId>org.apache.james</groupId>
         <artifactId>james-server-guice</artifactId>
         <version>3.6.0-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
+        <relativePath>../../pom.xml</relativePath>
```

Comment:
Does it makes sense to keep the blob- prefix for folders inside a blob directory? Idem for the guice suffix.

Shouldn't the naming scheme be guice/blob/api ?

Reply:
you are right, I wanted to minimize changes but should have included that bit.

Reply:
Should we do the same for the queue-xx-guice modules, if any?

Reply:
the queue-xxx- modules were already in *clean*  directories ( neither prefix nor suffix)